modules/new_serverpod_auth/serverpod_auth_session/serverpod_auth_session_server/lib/src/models/auth_success.spy.yaml

```diff
@@ -0,0 +1,11 @@
+class: AuthSuccess
```

Comment:
I wonder if the name could possibly be slightly different. Something like `AuthData` or simlar. Only reason I say this is because the existence of `AuthSuccess` might hint me that there also might be an `AuthError` or `AuthFailure` which is not the case.

This is definitely fine though. We don't need to change it. Just a thought

Reply:
Yeah, I am not entirely happy with it either, but also did not want to just do something super generic like `AuthInfo`.

Maybe it should be `AuthSession`, but then we just have to make the distinction between the internal and external representation clear (as the DB entity has the same name, but is not exposed).

It's also a bit weird now in the usage, as you also get an "AuthSuccess" upon password reset.

Luckily we don't have to commit to this quite yet, as we'll do a big "auth 2.0 review" at the end, going through all the introduced APIs (and packages) and align names and behavior across them. So maybe until then we have a grasp at a more suitable name.

Reply:
That makes sense :D Thanks for the clarification